CAIPs/caip-3.md

```diff
@@ -4,7 +4,7 @@ title: Blockchain Reference for the EIP155 Namespace
 author: Simon Warta (@webmaster128), ligi <ligi@ligi.de>, Pedro Gomes (@pedrouid)
 discussions-to: https://github.com/ChainAgnostic/CAIPs/issues/3, https://github.com/ChainAgnostic/CAIPs/pull/1
 status: Draft
```

Comment:
```suggestion
status: Superseded
```

Reply:
(Embarrassing! I'll cop to it, it was me that changed the wrong property to Superseded haha)

Reply:
if this is a real state we need to add it to CAIP-1